src/util/time.h

```diff
@@ -24,8 +24,14 @@ void UninterruptibleSleep(const std::chrono::microseconds& n);
  * This helper is used to convert durations before passing them over an
  * interface that doesn't support std::chrono (e.g. RPC, debug log, or the GUI)
  */
-inline int64_t count_seconds(std::chrono::seconds t) { return t.count(); }
-inline int64_t count_microseconds(std::chrono::microseconds t) { return t.count(); }
+constexpr inline int64_t count_seconds(std::chrono::seconds t) { return t.count(); }
+constexpr inline int64_t count_microseconds(std::chrono::microseconds t) { return t.count(); }
+
+/**
+ * Helper to count the seconds in any std::chrono::duration type
+ */
+using SecondsDouble = std::chrono::duration<double, std::ratio<1, 1>>;
+inline double CountSeconds(SecondsDouble t) { return t.count(); }
```

Comment:
nit: in the doxygen-generated documentation the comment will be assigned to `SecondsDouble` and `CountSeconds` will be left undocumented.

```cpp
/** These are the coordinates of the buzz point in the hyper plane. */
int x;
int y;
```
this looks ok in the source code but will produce unexpected result in the doxygen docs, leaving `y` undocumented.

Reply:
Thanks! Done.